ajax/libs/Repaintless.css/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "author": {
+    "email": "amjmigas@gmail.com",
+    "name": "Anna Migas"
+  },
+  "dependencies": {},
+  "description": "Repaintless is a library of CSS animations, which don't cause costly repaints and recalculations in our browser. Animations are smooth and simple.",
+  "homepage": "https://github.com/szynszyliszys/repaintless",
+  "keywords": [
+    "css",
+    "sass",
+    "scss",
+    "animations",
+    "performance",
+    "FLIP"
+  ],
+  "license": "MIT",
+  "name": "Repaintless.css",
+  "filename": "repaintless.min.css",
+  "optionalDependencies": {},
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/szynszyliszys/repaintless.git"
+  },
+  "version": "1.3.1",
+  "npmName": "repaintless",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "*.js",
+        "repaintless-css/repaintless*"
```

Comment:
If you use `"repaintless-css/repaintless*"`, you should make a "`repaintless-css`" folder for css files.